components/block/BlockDetails.tsx

```diff
@@ -67,13 +113,47 @@ const BlockDetails = () => {
   const { loading, error, data } = useQuery(GET_BLOCK_DETAILS, {
```

Comment:
Get error when rendering this page at this line.
```
Unhandled Runtime Error
Error: Rendered more hooks than during the previous render.
```

Reply:
Updated!